neo4j-examples/paradise-papers-django
# feature-13: create base for app

**NAME:**
[feature-13](https://trello.com/c/6uk9B7bj/13-as-a-developer-i-want-to-set-up-the-development-environment-so-that-i-can-start-development-process)

**DESCRIPTION:**
Create base for the application

**QA instruction:** 
1. clone project
2.  Go to paradise_papers_django folder
3. run python manage.py runserver
4. open http://127.0.0.1:8000/
5. see if this displays: 
![image](https://user-images.githubusercontent.com/29434543/33907408-7b74f60e-df5b-11e7-9311-da7d2cae5bc6.png)
